backend/tests/services/rag/test_runtime_specs.py

```
@@ -0,0 +1,86 @@
+        )
+
+
+@pytest.mark.parametrize(
+    ("kwargs", "expected_message"),
+    [
+        (
+            {"source_type": "attachment"},
+            "attachment_id is required",
+        ),
+        (
+            {"source_type": "attachment", "attachment_id": 123, "file_path": "/tmp/a"},
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/a"

(S108)